pkgs/development/python-modules/hebg/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # Runtime dependencies
+  networkx,
+  matplotlib,
+  numpy,
+  tqdm,
+  scipy,
+
+  # Build, dev and test dependencies
+  git,
+  setuptools-scm,
+  pytestCheckHook,
+  pytest-check,
+  pytest-mock,
+  sphinx,
+}:
+
+buildPythonPackage rec {
+  pname = "hebg";
+  version = "0.2.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-11bz+FbnaEVLiXT1eujMw8lvABlzVOeROOsdVgsyfjQ=";
+  };
+
+  build-system = [
+    git
```

Comment:
Do we really need git to build?

Reply:
I thought it was due to the setuptools-scm, but I just tested again and it works without it indeed